lib/auth/auth_with_roles.go

```diff
@@ -2740,7 +2740,7 @@ func (a *ServerWithRoles) GetUsers(ctx context.Context, withSecrets bool) ([]typ
 			}); err != nil {
 				log.WithError(err).Warn("Failed to emit local login failure event.")
 			}
-			return nil, trace.AccessDenied("User secret information requires admin rights and is not available from remote access")
+			return nil, trace.AccessDenied("reading User secrets requires admin rights and is not available from remote access")
```

Comment:
Users aren't going to understand that "requires admin rights" means "run locally on auth server". I think this needs to be improved.